### ISSUE TITLE
Peojwct doesn't install anything

### ISSUE THREAD

**Issue by yurivict** [steinbergmedia/vst3sdk#77]
It can't be packaged because it isn't clear what files should be installed.

**Comment by yurivict** [comment]
This is still a problem for 3.7.4_build_25

**Comment by yurivict** [comment]
@ygrabit 

Why is there no ```install``` target in this project?

**Comment by probonopd** [comment]
So, in other words, `sudo make install` is missing? Shouldn't be too hard for @steinbergmedia to fix?

Fixing this would improve VST3 support on FreeBSD it seems, among other things, so it'd be worthwhile I think.

**Comment by yurivict** [comment]
> So, in other words, sudo make install is missing?

Yes, they need to add cmake's ```install``` instructions, that's all.
